apisix/plugins/slslog/rfc5424.lua

```diff
@@ -81,7 +81,10 @@ local Severity = {
 local log_util = require("apisix.utils.log-util")
 
 
-local _M = { version = 0.1 }
+local _M = {
+    version = 0.1,
+    severity = Severity
```

Comment:
why we need to update this code?

Reply:
one PR for one thing. so I think we need to revert this PR

Reply:
Hi, thanks for the code review and the concern. The current PR need to access the rfc5424 severity map to decode alert status into integer numbers.
>    local message = {
        -- facility LOG_USER - random user level message
        "<".. tostring(8 + severity[conf.severity]) .. ">1", -- <PRIVAL>1
        timestamp,                                           -- timestamp
        ctx.var.host or "-",                                 -- hostname
        "apisix",                                            -- appname
        ctx.var.pid,                                         -- proc-id
        "-",                                                 -- msgid
        "[" .. conf.customer_token .. "@41058 " .. tab_concat(taglist, " ") .. "]",
        json_str
    }

I have made the field exported here from that module. Okay, reverting the changes back with a local copy of that severity map inside the plugin. Thank you